Core/FaviconsHelper.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  FaviconsHelper.swift
+//  DuckDuckGo
+//
+//  Copyright © 2022 DuckDuckGo. All rights reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import Foundation
+import Kingfisher
+
+struct FaviconsHelper {
```

Comment:
Heads up @Bunn in case I forget. I can't remember what you called your Favicon thing, but we might want to unify it and this when merging